src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -990,11 +990,13 @@ default boolean isEqual(@Nullable String other)
     }
 
     /**
-     * Compacts this Bytes object by moving the read position to the start.
+     * Compacts this Bytes object by moving the readPosition() to the start(),
+     * adjusting the readLimit()/writePosition() appropriately so that the readRemaining() is unchanged.
      * <p>
      * This operation is useful to free up space for writing by discarding bytes that have already
      * been read.
-     *
+     * <p>
+     * NOTE: If the space freed in longs is less than 1/4 the readRemaining in longs, this method does nothing to reduce movement overhead
```

Comment:
"longs" here is confusing

"If the space freed would be less than 1/4 of the amount to be moved, this method does nothing"